wordpress-mobile/WordPress-iOS
# Conditionally include HockeyApp for internal builds only

Fixes #5053 

To test:
Build for debug, archive for App Store and archive for Internal. Ensure no compilation errors and inspect bundles to check for HockeyApp artifacts. 

It does feel a little dirty having a couple conditionals based on a preprocessor macro.

Needs review: @sendhil, @koke
